YANG/ccamp/otn-tunnel/ietf-otn-tunnel.yang

```diff
@@ -697,38 +788,187 @@ module ietf-otn-tunnel {
+        + "te:path-in-segment/te:label-restrictions/"
+        + "te:label-restriction" {
+    description "OTN label.";
+    uses l1-types:otn-label-range-info;
+  }
+
+  /* Augment label restrictions start for the forwarding direction of
+     path-in-segment of secondary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-reverse-paths/te:secondary-reverse-path/"
+        + "te:path-in-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-start/"
+        + "te:te-label/te:technology" {
+    description "OTN label.";
+    case otn {
+      uses l1-types:otn-label-start-end;
+    }
+  }
+
+  /* Augment label restrictions end for the forwarding direction of
+     path-in-segment of secondary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-reverse-paths/te:secondary-reverse-path/"
+        + "te:path-in-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-end/"
+        + "te:te-label/te:technology" {
+    description "OTN label.";
+    case otn {
+      uses l1-types:otn-label-start-end;
+    }
+  }
+
+  /* Augment label-step of the label restrictions for the forwarding
+     direction of path-in-segment of secondary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-reverse-paths/te:secondary-reverse-path/"
+        + "te:path-in-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-step/te:technology" {
+    description "OTN label.";
+    case otn {
+      uses l1-types:otn-label-step;
+    }
+  }
+
+  /* Augment label restrictions for the forwarding direction of
+     path-out-segment of secondary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-reverse-paths/te:secondary-reverse-path/"
+        + "te:path-out-segment/te:label-restrictions/"
+        + "te:label-restriction" {
+    description "OTN label.";
+    uses l1-types:otn-label-range-info;
+  }
+
+  /* Augment label restrictions start for the forwarding direction of
+     path-out-segment of secondary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-reverse-paths/te:secondary-reverse-path/"
+        + "te:path-out-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-start/"
+        + "te:te-label/te:technology" {
+    description "OTN label.";
+    case otn {
+      uses l1-types:otn-label-start-end;
+    }
+  }
+
+  /* Augment label restrictions end for the forwarding direction of
+     path-out-segment of secondary reverse path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-reverse-paths/te:secondary-reverse-path/"
+        + "te:path-out-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-end/"
+        + "te:te-label/te:technology" {
+    description "OTN label.";
+    case otn {
+      uses l1-types:otn-label-start-end;
+    }
```

Comment:
Here it is correctly used otn-label-start-end grouping , why not above ?

Reply:
see comment above: in this case, the PR is adding augmentations for label-start, label-end and label-step.